doc/rst/02_getting_started.rst

```diff
@@ -0,0 +1,161 @@
+- Face the |vision board| horizontally, with the camera connector looking down.
+
+:numref:`Setup` shows the hardware setup for the example:
+
+.. _Setup:
+.. figure:: ../images/hw_setup.jpeg
+   :alt: Setup
+   :align: center
+   :width: 70%
+
+   Hardware Setup for the |vision board|
+
+Software Requirements
+---------------------
+
+The following software is required to build and run the library:
+
+- XTC tools: |xtc_tools_version| `XTC tools`_.
+- Python: |python_version| or later Python_.
+- CMake: |cmake_version| or later CMAKE_.
+
+Software Setup
+--------------
+
+Before building the example, ensure that the ``XTC tools`` are installed and properly activated in your development environment (`XTC tools`_). You can verify this by running the following command in your terminal:
+
+.. code-block:: console
+
+   xcc --version
+
+This command should display the version of the installed ``XTC tools``. If the tools are not installed or activated, refer to the installation instructions in the documentation.
+
+Once done, run then run the following commands from the root of the library:
+
+.. code-block:: console
+
+   # install python dependencies
+   pip install -r requirements.txt
+   # go to example folder
+   cd examples/capture_rgb
+   # build
+   cmake -G "Unix Makefiles" -B build
+   xmake -C build
+
+If the build is successful, the message ``[100%] Built target capture_rgb`` and the usage report will be displayed:
+
+.. list-table:: Memory Usage Per Tile
+   :header-rows: 1
+   :align: left
+
+   * - Tile
+     - Memory Used
+     - Status
+   * - tile[0]
+     - 4236 / 524288
+     - OKAY
+   * - tile[1]
+     - 170692 / 524288
+     - OKAY
+
+Note that the memory usage is shown in bytes, and results could differ slightly. The maximum memory available for each tile is 524288 bytes. As we can see in this case, the memory usage is well within the limits for both tiles. Usage is about 0.81% of the total memory available for tile 0 and about 32% for tile 1. 
+
+Once the example is built, you can run it using the following command:
+
+.. code-block:: console
+
+   # run 
+   xrun --xscope bin/capture_rgb.xe
+   # decode image
+   python decode.py
+
+This will run the example, save the image to a binary file, and then the python script will decode the image and display it.
+
+To go further, uncomment the following lines in the ``user_app.c`` and re-run the file:
+
+.. literalinclude:: ../../examples/capture_rgb/src/user_app.c
+   :language: c
+   :start-at:  /* (Optional) try something out of bounds
+   :end-before: exit(0);
+
+This will try to apply an offset > 1.0 and will cause an error. The error will be displayed in the terminal, and the program will exit with an error code.
+
+.. code-block:: console
+
+   xrun: Program received signal ET_ECALL, Application exception.
+   0x00081ad6 in camera_isp_coordinates_compute (img_cfg=0xe2a2c) at ...camera_isp.c:216
+
+Line 216 is the line where the error is raised. The line expects the following:
```

Comment:
Should we say something like "like 216 is the line exception happened in this case", I'm sure the source code will change in the future and it won't be 216 at some point